tdp/cli/commands/deploy.py

```diff
@@ -89,17 +89,21 @@ def deploy(
         ).run(planned_deployment, force_stale_update=force_stale_update)
 
         if dry:
-            for _ in deployment_iterator:
-                if _:
-                    _()
+            for op, fn in deployment_iterator:
+                if fn:
+                    fn(op)
+                click.echo(op)
             return
 
         # deployment and operations records are mutated by the iterator so we need to
         # commit them before iterating and at each iteration
         session.commit()  # Update operation status to RUNNING
-        for process_operation_fn in deployment_iterator:
+        for operation_rec, process_operation_fn in deployment_iterator:
             session.commit()  # Update deployment and current operation status to RUNNING and next operations to PENDING
-            if process_operation_fn and (cluster_status_logs := process_operation_fn()):
+            if process_operation_fn and (
+                cluster_status_logs := process_operation_fn(operation_rec)
+            ):
+                click.echo(operation_rec)
```

Comment:
We may want to display a meaningful message here.

Reply:
operation_rec already prints so much parameters which the user can check afterwards in the DB and adding extra texts will print to much output in terminal.

I propose only displaying the name of the operation and its state.
```py
f"Operation {operation_rec.operation} is in state {operation_rec.state}"
```